src/constfold.d

```diff
@@ -450,12 +449,12 @@ extern (C++) UnionExp Mod(Loc loc, Type type, Expression e1, Expression e2)
         if (e2.type.isreal())
         {
             real_t r2 = e2.toReal();
-            c = complex_t(Port.fmodl(e1.toReal(), r2), Port.fmodl(e1.toImaginary(), r2));
+            c = complex_t(e1.toReal() % r2, e1.toImaginary() % r2);
```

Comment:
What? Let's require `real_t` to support the modulo operator instead of going through `Port`...